### ISSUE TITLE
Build issue in scriptmanager.cpp

### ISSUE THREAD

**Issue by kyleaolson** [easymodo/qimgv#59]
Hello!
I'm attempting to install on Ubuntu 18.04 and am running into an issue when building. I have installed the required dependencies per the instructions in the README. Any thoughts on what might be causing this? I'm not familiar with C++ enough to really parse through the error.

Here is the steps I took and the output of the ./build.sh script.
https://gist.github.com/kyleaolson/c9877cbe030457b3c68414f09ea5ea0d

Thanks!

**Comment by kamiyaa** [comment]
I believe your dependencies does not meet the minimum version.
More specifically:
```
>=dev-qt/qtconcurrent-5.10.0
>=dev-qt/qtcore-5.10.0
>=dev-qt/qtgui-5.10.0
>=dev-qt/qtwidgets-5.10.0
```

**Comment by easymodo** [comment]
Thanks for reporting!
Pushed a fix, it should now build with qt 5.9

**Comment by kyleaolson** [comment]
Was able to build/install after pulling the latest. Thanks @easymodo !
